### ISSUE TITLE
Feat: improve the user experience of member cluster console

### ISSUE THREAD

**Issue by wansir** [kubesphere/kubesphere#4610]
**What's it about?**

Improve the user experience and render the console according to different cluster roles.

**What's the reason why we need it?**

Currently, the console of the member cluster is not stable, the incorrect operation will cause data loss or resource conflict.

Here is a list of things we wish to do.

- [x] User/Workspace is managed by kubefed in host cluster, user and workspace management needs to be disabled in member cluster's console.
- [x] Currently, it is not possible to enter workspace from the member cluster due to API differences.
- [x] Pay attention to federated resources and global resources to avoid conflicts.

**Area Suggestion**

 /area console
/kind feature-request

**Comment by harrisonliu5** [comment]
By judging the member cluster to shield the access control platform role creation operation and the creation operation in the enterprise space setting.

**Comment by iawia002** [comment]
I will add the role of the cluster into the `kubesphere-config` configmap:

```diff
multicluster:
  enable: true
  agentImage: kubesphere/tower:v0.2.0
  proxyPublishService: tower.kubesphere-system.svc
  proxyPublishAddress: http://103.61.38.60:30361
+ clusterRole: host/member
```

**Comment by wenxin-01** [comment]
/assign @harrisonliu5 
There is no workspace created in host cluster on the console page of member cluster.

host-cluster console: 
`/kapis/tenant.kubesphere.io/v1alpha2/workspaces?sortBy=createTime&limit=10`
![image](https://user-images.githubusercontent.com/68640256/168262077-dc6d2dfe-bc24-4e14-b471-7cdf39acd950.png)

member-cluster console:
`/kapis/tenant.kubesphere.io/v1alpha2/workspaces?sortBy=createTime&limit=10`
![image](https://user-images.githubusercontent.com/68640256/168262169-1ca98134-4924-4116-9cda-714d0b0c5b10.png)

**Comment by wenxin-01** [comment]
Issue fixed in `kubespheredev/ks-installer:nightly-20220524`. But still have some issue in member-cluster console.
https://github.com/kubesphere/kubesphere/issues/4913
